src/Symfony/Component/Validator/Constraints/Locale.php

```diff
@@ -29,4 +29,13 @@ class Locale extends Constraint
 
     public $message = 'This value is not a valid locale.';
     public $canonicalize = false;
+
+    public function __construct($options = null)
+    {
+        if (!($options['canonicalize'] ?? false)) {
+            @trigger_error('"canonicalize" option with value `false` is deprecated since Symfony 4.1 and will be removed in 5.0. Use "canonicalize"=>true instead.', E_USER_DEPRECATED);
```

Comment:
The .... Set it to `true` instead.

Reply:
I took as example the syntax at [`Email::__construct()`](https://github.com/symfony/symfony/blob/3ef76bfc0b24df72940b1691617844d5e829b33f/src/Symfony/Component/Validator/Constraints/Email.php#L62). Updated.